### PR TITLE
Fix gajim.profile

### PR DESCRIPTION
**Pull request by rusty-snake**
On my system (F29) gajim only starts if it has access to the user default shell.

### CODE REVIEW

**Comment by rusty-snake** [comment]
Gajim also needs access to `paplay` to play notification tones.

**Comment by rusty-snake** [comment]
Don't merge yet, I think I'll have a few more fixes in the next few days.

File: etc/gajim.profile [hunk 1]
```diff
@@ -41,9 +41,15 @@ notv
 nou2f
 protocol unix,inet,inet6
 seccomp
+tracelog
+shell none
```
Comment (SkewedZeppelin) [inline etc/gajim.profile]: swap

Comment (rusty-snake) [reply etc/gajim.profile]: OK. Done.

File: etc/gajim.profile [hunk 2]
```diff
@@ -41,9 +41,15 @@ notv
...
 
 disable-mnt
-private-bin python,python3,sh,gpg,gpg2,gajim
+private-bin python,python3,sh,gpg,gpg2,gajim,bash,zsh,paplay,gajim-history-manager
+private-cache
 private-dev
 private-etc alsa,asound.conf,ca-certificates,crypto-policies,fonts,group,hostname,hosts,ld.so.cache,ld.so.conf,localtime,machine-id,passwd,pki,pulse,resolv.conf,ssl
 private-tmp
+
+noexec ${HOME}
```
Comment (Fred-Barclay) [inline etc/gajim.profile]: Will `noxec ${HOME}` prevent plugins from running?

Comment (rusty-snake) [reply etc/gajim.profile]: Short: No.
Long: I was thinking, too, but I didn't notice any mistakes. I checked and all plugin files in `.local/share/gajim` have `rw-r-----`. All plugins are written in python.

**Comment by SkewedZeppelin** [comment]
Merged, thanks!

**Pull request merged**